improver/standardise.py

```diff
@@ -140,7 +257,10 @@ def __init__(
         Raises:
             ValueError: If a landmask is required but not passed in
         """
-        if landmask is None and "nearest-with-mask" in regrid_mode:
+        if regrid_mode not in self.REGRID_REQUIRES_LANDMASK:
+            msg = "Unrecognised regrid mode {}"
+            raise ValueError(msg.format(regrid_mode))
```

Comment:
I think this new `ValueError` can be added to the `Raises` section above.

Reply:
Ah you are right the section is now incomplete.  I actually don't think this section adds anything useful, and a `Raises` section isn't compulsory from our style guide, so I've removed it from this docstring.